web/src/user_group_edit_members.ts

```diff
@@ -229,10 +266,18 @@ function add_new_members({pill_user_ids}) {
         });
     }
 
-    function invite_failure(xhr) {
+    function invite_failure(xhr?: JQuery.jqXHR): void {
         let message = "Failed to add user!";
```

Comment:
This can be a follow-up, but the logic with this string is definitely wrong, since it's not tagged for translation. Might be worth a #frontend thread as to whether there's a more generic thing to do; I think we have an `xhr_error` shared method that I'm not sure why we're not using here.